packages/suite-desktop/src-electron/modules/tor.ts

```diff
@@ -99,6 +99,11 @@ const init = async ({ mainWindow, store }: Dependencies) => {
 
         cb({ cancel: false });
     });
+
+    app.on('before-quit', () => {
+        // fix: kill tor process on app close
+        toggleTor(false);
```

Comment:
This will turn off Tor for any platform and save it in the settings. The next launch will have Tor disabled which is not the desired behaviour. 

The best place to look at imo would be the `stop` method in `packages/suite-desktop/src-electron/libs/processes/BaseProcess.ts`. There should be a reason why killing the process failed and maybe there should be a way to force this on Mac.